### PR TITLE
Temporarily disable problematic ocean test

### PR DESCRIPTION
**Pull request by mihails-strasuns**

### CODE REVIEW

File: vars/runPipeline.groovy [hunk 1]
```diff
@@ -211,6 +211,7 @@ def testDownstreamProject (name) {
 
                 case 'sociomantic-tsunami/ocean':
                     sh '''
+                    echo '\nTEST_FILTER_OUT += $C/test/collectd/main.d' >> Build.mak
```
Comment (nemanja-boric-sociomantic) [inline vars/runPipeline.groovy]: This is the wrong test ;-). Should be `test/sysstats/main.d`.

**Comment by mihails-strasuns** [comment]
Green

**Comment by MartinNowak** [comment]
Think I found the culprit for the random test failures, https://github.com/sociomantic-tsunami/ocean/pull/389.

**Comment by nemanja-boric-sociomantic** [comment]
https://github.com/dlang/ci/pull/88#issuecomment-347979836 :-)

**Pull request merged**